src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -375,15 +376,27 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
       );
     },
     renderTopToolbarCustomActions: ({ table }) => (
-      <Button
-        variant="outlined"
-        onClick={() => {
-          setItemsDialogType('create');
-          table.setCreatingRow(true);
-        }}
-      >
-        Add Catalogue Item
-      </Button>
+      <Box sx={{ display: 'flex' }}>
```

Comment:
Just wondering if we may want to do it like datagateway, disabling if there are no filters/having icons in them. 
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/9454e509-b151-443d-80b1-91ab8d0f5154)
I don't mind either way, just thought I would mention it.

Reply:
Should I add the add icon to the add catalogue item and category then ??

Reply:
Yes I think so, then its the same as move/copy as well.